sky/optimizer.py

```diff
@@ -827,10 +827,7 @@ def _fill_in_launchable_resources(
 ) -> Tuple[Dict[resources_lib.Resources, List[resources_lib.Resources]],
            _PerCloudCandidates]:
     enabled_clouds = global_user_state.get_enabled_clouds()
-    if backend_utils.is_public_cloud_disabled() and try_fix_with_sky_check:
```

Comment:
Should we be concerned with `try_fix_with_sky_check` not being used in the new change?